java/maven/apichanges.xml

```diff
@@ -83,6 +83,49 @@ is the proper place.
     <!-- ACTUAL CHANGES BEGIN HERE: -->
 
     <changes>
+        <change id="ProjectLookup.fallback">
+            <api name="general"/>
+            <summary>Allow project services, that are ordered <b>after</b> specific packaging type</summary>
+            <version major="2" minor="144"/>
+            <date day="10" month="2" year="2021"/>
+            <author login="sdedic"/>
+            <compatibility addition="yes" binary="compatible" semantic="compatible"/>
+            <description>
+                <p>
+                    Project lookup consists of base services, plus services for the project's packaging type. This change allows for
```

Comment:
I would expect a note describing how to register such service! Maybe a link to `architecture-summary.html#xyz`.

Reply:
Documented in 9d7654ce75